### ISSUE TITLE
0.8.1 keeps writing into "version_0" folder instead of creating new version_1/2/3... 

### ISSUE THREAD

**Issue by AAnoosheh** [PyTorchLightning/pytorch-lightning#2761]
0.7.6 (I believe) would properly create a new "version_X" folder per run, but since upgrading to 0.8.1, it no longer does this.

Here's my logging-related code in my train script, which are then passed onto Trainer:
```
    # custom logging directory
    logger = pl.loggers.TestTubeLogger(
        save_dir=logging_dir,
        name=args.name
    )
    log_ckpt = pl.callbacks.ModelCheckpoint(save_top_k=-1, verbose=True)
```

**Comment by rohitgr7** [comment]
Try 0.8.5?

**Comment by awaelchli** [comment]
or you may even need to be on master (or 0.9 soon). I recently fixed an issue with save_dir here #2681

**Comment by Borda** [comment]
@AAnoosheh I am sorry, there won't be any fixes in v0.8.1 pls consider an upgrade to latest version
